js/views/transactionsVw.js

```diff
@@ -399,12 +398,10 @@ module.exports = pageVw.extend({
 
     return $.when.apply(null, calls)
         .fail(function(){
-          self.registerChild(
-            new Dialog({
```

Comment:
@rmisio this was throwing an error after I merged because Dialog doesn't exist. It should just be using app.simpleMessageModal, correct?